frontend/packages/kubevirt-plugin/src/components/create-vm/forms/boot-source-form-reducer.ts

```diff
@@ -107,7 +106,8 @@ export const bootFormReducer = (
   };
   let isValid: boolean;
   const sizeValidation = validatePositiveInteger(`${newState.size?.value.value}`, {
-    subject: `${PersistentVolumeClaimModel.label} size`,
+    // t('kubevirt-plugin~Persistent Volume Claim size')
+    subject: `kubevirt-plugin~Persistent Volume Claim size`,
```

Comment:
does it make sense here to replace `subject` with `subjectKey` to make sure users know it need translation when using it ?

Reply:
actually the whole validate* logic is constructing sentences which is not good. We will have to rework it. I've removed the translation for now and we will need to address it it separate PR.